src/prog_models/models/uav_model/uav_model.py

```diff
@@ -0,0 +1,556 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+from prog_models.prognostics_model import PrognosticsModel
+import prog_models.models.uav_model.trajectory.route as route 
+import prog_models.models.uav_model.trajectory.trajectory as trajectory
+from prog_models.models.uav_model.vehicles import AircraftModels
+from prog_models.models.uav_model.utilities import geometry
+
+import numpy as np
+import datetime
+from copy import deepcopy
+import prog_models.models.uav_model.utilities.geometry as geom
+from warnings import warn
+from prog_models.exceptions import ProgModelInputException
+
+class UAVGen(PrognosticsModel):
+    """
+    Vectorized prognostics :term:`model` to generate a predicted trajectory for a UAV. 
+    
+    Model generates cartesian positions and velocities, pitch, roll, and yaw, and angular velocities throughout time to satisfy some user-define waypoints. 
+
+    :term:`Events<event>`: (1)
+        TrajectoryComplete: percentage of waypoints remaining for trajectory to pass through
+    
+    :term:`Inputs/Loading<input>`: ()
+        | User-defined inputs: waypoints plus ETAs or speeds
+        | Model-defined inputs: 
+            | T: thrust
+            | mx: moment in x 
+            | my: moment in y
+            | mz: moment in z
+
+    :term:`States<state>`: (13)
+        | x: cartesian position
+        | y: cartesian position
+        | z: cartesian position
+        | phi: pitch
+        | theta: roll
+        | psi: yaw
+        | vx: velocity 
+        | vy: velocity 
+        | vz: velocity 
+        | p: angular velocity 
+        | q: angular velocity 
+        | r: angular velocity 
+        | t: time 
+
+    :term:`Outputs<output>`: (12)
+        | x: cartesian position
+        | y: cartesian position
+        | z: cartesian position
+        | phi: pitch
+        | theta: roll
+        | psi: yaw
+        | vx: velocity 
+        | vy: velocity 
+        | vz: velocity 
+        | p: angular velocity 
+        | q: angular velocity 
+        | r: angular velocity 
+        | t: time 
```

Comment:
```suggestion
```

Time isn't an output, right? It's not in model.outputs, below

Reply:
I think this one was missed. I still see output here, but it's not in outputs on line 154